plugins/mysql/CMakeLists.txt

```diff
@@ -58,7 +60,9 @@ if(MYSQLEMBED)
             ${MYSQL_LIBRARIES}
             eclrtl
             roxiemem
-            jlib)
+            jlib
```

Comment:
This looks like it would affect non-vcpkg builds (for example)

Reply:
Not really - its just moving the find_package logic out of the macro (which is bit buggy).  The addition of the  ZLIB is simply being more specific due to USE_OPTIONAL "magic".

Reply:
IOW in non vcpkg build with USE_PLUGINS=1 and USE_OPTIONAL=1 and ZLIB is not installed, the build will fail.